### PR TITLE
Added support for SFRPG

### PR DESCRIPTION
**Pull request by Yukonite**
Hi! I believe I've added support for SFRPG, though I'll need to review in Foundry to confirm. Thanks for building this module!

### CODE REVIEW

File: module.json [hunk 1]
```diff
@@ -19,10 +19,10 @@
         "pf1",
         "pf2e",
         "D35E",
-        "sw5e"
+        "sw5e",
+        "sfrpg"
     ],
-    "languages": [
-        {
+    "languages": [{
```
Comment (Varriount) [inline module.json:25]: The change on these two lines needs to be reverted (slight formatting inconsistency).
Same for line 66 and 88.

File: src/lmrtfy.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ class LMRTFY {
             onChange: () => window.location.reload()
         });
 
-        Handlebars.registerHelper('lmrtfy-controlledToken', function (actor) {
+        Handlebars.registerHelper('lmrtfy-controlledToken', function(actor) {
```
Comment (Varriount) [inline src/lmrtfy.js:22]:
```suggestion
        Handlebars.registerHelper('lmrtfy-controlledToken', function (actor) {
```

File: src/lmrtfy.js [hunk 3]
```diff
@@ -130,7 +143,7 @@ class LMRTFY {
             LMRTFY.requestor.options.classes.push("lmrtfy-parchment")
         else
             LMRTFY.requestor.options.classes = LMRTFY.requestor.options.classes.filter(c => c !== "lmrtfy-parchment")
-        // Resize to fit the new theme
+            // Resize to fit the new theme
```
Comment (Varriount) [inline src/lmrtfy.js:146]:
```suggestion
        // Resize to fit the new theme
```

File: src/lmrtfy.js [hunk 4]
```diff
@@ -166,4 +179,4 @@ class LMRTFY {
 Hooks.once('init', LMRTFY.init);
 Hooks.on('ready', LMRTFY.ready);
 Hooks.on('getSceneControlButtons', LMRTFY.getSceneControlButtons);
-Hooks.on('renderChatMessage', LMRTFY.hideBlind);
+Hooks.on('renderChatMessage', LMRTFY.hideBlind);
```
Comment (Varriount) [inline src/lmrtfy.js:182]: Please add a newline to the end of the file.

**Review by Varriount — CHANGES_REQUESTED** [review]

**Comment by Varriount** [comment]
The PR looks good, you mainly need to revert some formatting changes that occurred. Please comment when you have had a chance to test your changes in Foundry.

**Comment by vtt-lair** [comment]
@Yukonite can you possible supply me with the "mod" name for the modifier of your actor's ability rolls? In other words if you roll a strength ability check, what variable is used? For instance in DnD5E it's "@abilities.dex.mod". 

This is for my PR #106 that will bring a bit more functionality to the custom formula field. Just so that I can add them, as your PR will probably be going first (and would make more sense for me to wait for yours to be merged in first)

**Comment by Yukonite** [comment]
@vtt-lair The formatting is the same as noted above - "@abilities.dex.mod"

File: src/lmrtfy.js (hunk 3 above)
Comment (Yukonite) [reply src/lmrtfy.js]: Changed.

**Comment by Yukonite** [comment]
@Varriount Tested in Foundry. The only skill check that won't work with the existing code is Initiative because the attribute value is 'init' not 'initiative' in the SFRPG system. I can try a workaround with something like:

`LMRTFY.specialRolls = { 'init': true };` or `LMRTFY.specialRolls = { 'SFRPGinit': true };`

and then make adjustments to the HTML, roller, and requestor. Thoughts?

**Comment by Varriount** [comment]
@Yukonite I'm unfamiliar with SFRPG (and also LMRTFY - I just review general PRs for https://github.com/League-of-Foundry-Developers ). Regular initiative rolls add the user's selected token to the combat tracker, if there is a combat available. I would recommend installing another system and testing in that one (Pathfinder or D&D 5e both have an "initiative" concept).

**Comment by vtt-lair** [comment]
@Yukonite if you pull the latest to your branch you'll have the added dice feature below the custom formula field. To make sure it works for SFRPG too, please add the following 3 lines to your case:

`
LMRTFY.abilityAbbreviations = CONFIG.SFRPG.abilityAbbreviations;
LMRTFY.modIdentifier = 'mod';
LMRTFY.abilityModifiers = LMRTFY.parseAbilityModifiers();
`

Then regarding your initiative question. I quickly tried it on a DnD5e system to see what happens... and from what I can see currently the initiative and death saving throw doesn't seem to be working. I'll try and spend some time over the weekend to figure out what is going and and why it's not working... I don't really use LMRTFY's initiative or death saves feature in my games.

**Comment by Yukonite** [comment]
Old pull request. I may update when I get LMRTFY working with SFRPG again.

**Pull request closed without merging**